src/manifests/test_manifest.py

```diff
@@ -62,6 +62,19 @@ class TestManifest(ComponentManifest['TestManifest', 'TestComponents']):
                         "type": "dict",
                         "schema": {
                             "build-dependencies": {"type": "list"},
+                            "topology": {
+                                "type": "list", "required": False,
+                                "schema": {
+                                    "type": "dict",
+                                    "schema": {
+                                        "cluster_name": {"required": True, "type": "string"},
+                                        "data_nodes": {"type": "integer", "required": True},
+                                        "master_nodes": {"type": "integer"}
```

Comment:
The master terminology is deprecated in 3.0, any way we can make this `cluster_manager_nodes` now?

Reply:
Sure, made the changes.